src/domain/registration/welcome/Welcome.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import tadaImage from '../../../assets/icons/svg/tada.svg';
+import Button from '../../../common/components/button/Button';
+import Icon from '../../../common/components/icon/Icon';
+import styles from './welcome.module.scss';
+
+const Welcome: FunctionComponent = () => {
+  const { t } = useTranslation();
+  return (
+    <div className={styles.welcome}>
+      <div className={styles.hero}>
+        <div className={styles.heroContainer}>
+          <h1>{t('registration.welcome.hero.header')}</h1>
```

Comment:
you dont need that much divs and reused styles
1 wrapper div is ok
this screen deserve fullscreen

Reply:
good point!